test_sets/new_vehicle_data_hands_off_steering.txt

```diff
@@ -0,0 +1,35 @@
+./test_scripts/API/VehicleData/HandsOffSteering/001_GetVD_Success_flow.lua
+./test_scripts/API/VehicleData/HandsOffSteering/002_GetVD_disallowed.lua
+./test_scripts/API/VehicleData/HandsOffSteering/003_GetVD_disallowed_after_PTU.lua
+./test_scripts/API/VehicleData/HandsOffSteering/004_GetVD_HMI_does_not_respond.lua
```

Comment:
@GetmanetsIrina Set needs to be updated since it includes non existing scripts (e.g. 004 - 008)